frontend/src/context/layers/composite_data.ts

```diff
@@ -13,26 +13,25 @@ export const fetchCompositeLayerData: LazyLoader<CompositeLayerProps> = () => as
   params: LayerDataParams<CompositeLayerProps>,
   { dispatch },
 ) => {
-  // to complete later with new endpoint for composite chart
-
   const { layer, date } = params;
-  const endDate = (date ? new Date(date) : new Date())
-    .toISOString()
-    .split('T')[0];
-  const { baseUrl, inputLayers, startDate } = layer;
+  const startDate = date ? new Date(date) : new Date();
```

Comment:
To be discussed but my first guess would have been to do the opposite and use the selectedDate as the endDate and then startDate = selectedDate - 1month

Reply:
This is actually the way it's done usually in HDC apparently so let's keep it like that.